### PR TITLE
adds change

### PR DESCRIPTION
**Pull request by Zhang**

### CODE REVIEW

**Comment by maxlang** [comment]
I like it

File: Readme.md [hunk 1]
```diff
@@ -82,6 +82,40 @@ if (true)
...
+## Pass all optional parameters as a single object
+
+Pass all optional parameters as a single object, it makes it easier to change function
+definitions.
+
+*Right:*
+
+```js
+function foo(bar, opts) {
+  this.color = opts.color;
```
Comment (danverbraganza) [inline Readme.md]: Recommend using _.defaults to both ensures defaults, and documents it in the code with strong locality.

opts = _.defaults(opts, {
    colour: 'red',
    speed: '250m.s-1',
    sound: 'wooo'
  })

**Comment by ianlyons** [comment]
i like both the original suggestion and danver's addition

**Comment by eugeneiiim** [comment]
LGTM contingent on Danver's comment.

**Pull request merged**